packages/blueprints/subscribe-blueprint/README.md

```diff
@@ -1,17 +1,19 @@
 # Subscribe to External Blueprint
 
-This blueprint allows you to subscribe to community blueprint packages distributed on NPM. It creates workflows that periodically import public NPM
-packages as new private blueprints in your space. This requires that the target blueprint be avaiable through NPM.
+This blueprint allows you to subscribe to community blueprint packages on NPM. The generated workflows periodically pull NPM packages and publishes
+them as custom blueprints in your CodeCatalyst space. A workflow is created for each package you want imported, so the target blueprint that you want
```

Comment:
I think you mean either
```suggestion
them as custom blueprints in your CodeCatalyst space. A workflow is created for each package you import, so the target blueprint that you want
```

or

```suggestion
them as custom blueprints in your CodeCatalyst space. A workflow is created for each package you want to import, so the target blueprint that you want
```

?

Reply:
Yeah good idea!